test/fixtures/es-class-extend-deopt/expected-remove-es6.js

```diff
@@ -2,7 +2,14 @@ import BaseComponent from 'components/base';
 
 class Foo extends BaseComponent {
   render() {}
+
 }
-Foo.propTypes = {
-  foo: PropTypes.string.isRequired
-};
\ No newline at end of file
+
+Object.defineProperty(Foo, "propTypes", {
```

Comment:
Broken test? Shouldn't it remove it?

Reply:
How is this broken?

Reply:
It should remove prop types, isn't it? Am I not noticing something obvious?

Reply:
It shouldn't. It's here to show a limitation of the lib. It's a tradeoff taken for perf.

Reply:
I see, that's why it's not working for me as I expect it to. Looks like a bug to be tho.

Reply:
@vlad-zhukov It's related to #8